vector-app/src/androidTest/java/im/vector/app/CantVerifyTest.kt

```diff
@@ -34,6 +35,7 @@ import java.util.UUID
 
 @RunWith(AndroidJUnit4::class)
 @LargeTest
+@Ignore // Temporary ignore whilst confirming other tests
```

Comment:
I'm currently investigating re-enabling this and the `VoiceRecorderLTests` tests

Reply:
`CantVerifyTest` is fixed by https://github.com/vector-im/element-android/pull/7210
`VoiceRecorderLTests` is fixed by https://github.com/vector-im/element-android/pull/7226